packages/service-library/src/servicelib/aiohttp/application_setup.py

```diff
@@ -114,11 +140,13 @@ def app_module_setup(
     module_name: str,
     category: ModuleCategory,
     *,
+    # TODO: use settings_name as module_name!!
+    settings_name: Optional[str] = None,
     depends: Optional[List[str]] = None,
+    logger: logging.Logger = log,
+    # TODO: deprecate these options
```

Comment:
is that still todo? or forgotten?

Reply:
same as above. I added more doc in the TODO